workflows/Post_Exchange_Phishing_Alerts_to_Microsoft_Teams/workflow.spec.yaml

```
@@ -3,13 +3,13 @@ products: ["insightconnect"]
 name: Post_Exchange_Phishing_Alerts_to_Microsoft_Teams
 title: "Post Exchange Phishing Alerts to Microsoft Teams"
 description: "This workflow will dissect an email and return all potential malicious indicators to Microsoft Teams in a message."
-version: 1.0.3
+version: 1.0.4
 vendor: rapid7
 support: rapid7
 status: []
 hub_tags:
   use_cases: [threat_detection_and_response]
-  keywords: [phishing, email, microsoft, work_from_home, exchange]
+  keywords: [phishing, email, microsoft_teams, Exchange]
```

Do we want Exchange to be capitalized?
```suggestion
  keywords: [phishing, email, microsoft_teams, exchange]
```